xbmc/cores/VideoPlayer/DVDCodecs/Video/AMLCodec.cpp

```diff
@@ -2660,21 +2661,6 @@ CDVDVideoCodec::VCReturn CAMLCodec::GetPicture(VideoPicture *pVideoPicture)
     CLog::Log(LOGDEBUG, LOGVIDEO, "CAMLCodec::GetPicture: index: {:d}, pts: {:.3f}, dur:{:.3f}ms ar:{:.2f} elf:{:d}ms",
       m_bufferIndex, pVideoPicture->pts / DVD_TIME_BASE, pVideoPicture->iDuration / 1000, m_hints.aspect, elapsed_since_last_frame.count());
 
-    if (m_dolby_vision_wait_delay > 0 && !m_is_dv_p7_mel)
-    {
-      m_dolby_vision_wait_delay--;
-      CSysfsPath is_mel{"/sys/module/amdolby_vision/parameters/is_mel"};
-      if (is_mel.Exists())
-      {
-        if (is_mel.Get<char>().value() == 'Y')
-        {
-          CLog::Log(LOGDEBUG, LOGVIDEO, "CAMLCodec::GetPicture: DoVi P{:d} MEL content detected, request to reopen decoder",
-            m_hints.dovi.dv_profile);
-          m_is_dv_p7_mel = true;
-          return CDVDVideoCodec::VC_REOPEN;
```

Comment:
How do we decide when to reopen the decoder with this removed and no new logic in its stead?

Reply:
The reopen here is just because of the detected MEL case - no longer need to reopen because of MEL is know up front.